source/docs/3/nearest_neighbor_analysis.rst

```diff
@@ -34,103 +39,74 @@ Procedure
 
-  .. image:: /static/3/nearest_neighbor_analysis/images/2.png
+  .. image:: /static/3/nearest_neighbor_analysis/images/02.png
      :align: center
 
-3. Click the :guilabel:`...` button next to :guilabel:`File name` and browse to the downloaded ``signif.txt`` file. Once loaded, the :guilabel:`File Format` and :guilabel:`Geometry Definition` fields should be auto-populated with correct values. Click :guilabel:`Add` followed by :guilabel:`Close`.
+3. In the :guilabel:`Data Source Manager` dialog box, select :guilabel:`Delimited Text`. 
 
-  .. image:: /static/3/nearest_neighbor_analysis/images/3.png
+  .. image:: /static/3/nearest_neighbor_analysis/images/03.png
      :align: center
 
-4. Zoom around and explore both the datasets. Each yellow point represents the location of a significant earthquake and each red point represents the location of a populated place. Our goal is to find out the nearest point from the populated places layer for each of the points in the earthquake layer.
+4. Click the :guilabel:`...` button next to :guilabel:`File name` and browse to the downloaded ``earthquakes-2021-11-25_13-39-30_+0530.tsv`` file. Once loaded, the :guilabel:`File Format` and :guilabel:`Geometry Definition` fields should be auto-populated with correct values. Cross verify it with the markings below if required update field acordingly. Click :guilabel:`Add` followed by :guilabel:`Close`.
 
-  .. image:: /static/3/nearest_neighbor_analysis/images/4.png
+  .. image:: /static/3/nearest_neighbor_analysis/images/04.png
      :align: center
 
-5. Before we do the analysis, we need to clean up our inputs. The ``signif`` layer contains many records without a valid geometry. These records were imported with a **NULL** geometry. So let's remove these records first. Go to :menuselection:`Processing --> Toolbox`.
+5. Zoom around and explore both datasets. Each red point represents the location of an earthquake incidence, and each green point represents the location of a populated place. Our goal is to find out the nearest point from the populated places layer for each of the points in the earthquake layer. Let's inspect the Attribute table of the earthquakes layer. Select the layer and click on :guilabel:`Open Attribute Table` icon in :guilabel:`Toolbar`. 
 
-  .. image:: /static/3/nearest_neighbor_analysis/images/5.png
+  .. image:: /static/3/nearest_neighbor_analysis/images/05.png
      :align: center
 
-6. Search for and locate the :menuselection:`Vector geometry --> Remove null geometries` tool. Double-click to launch it.
+6.  There are ``2586`` features, but the data contains few entries with invalid geometry. We have to remove that before proceeding further. Close the Attribute Table. 
```

Comment:
instead of invalid geometry, mention "data contains some records with no latitude or longitude infomation".

Reply:
done.